eox_tenant/backends/database.py

```diff
@@ -151,3 +160,29 @@ def _set_microsite_config_from_obj(self, subdomain, domain, microsite_object):
         config['site_domain'] = strip_port_from_host(domain)
         config['microsite_config_key'] = microsite_object.key
         self.current_request_configuration.data = config
+
+    def enable_microsites(self, log):
+        """
+        Configure the paths for the microsites feature
+        """
+        microsites_root = settings.MICROSITE_ROOT_DIR
+
+        if os.path.isdir(microsites_root):
+            settings.STATICFILES_DIRS.insert(0, microsites_root)
+            settings.LOCALE_PATHS = [microsites_root / 'conf/locale'] + settings.LOCALE_PATHS
+
+            log.info('Eox-tenant is loading microsite path at %s', microsites_root)
+        else:
+            log.error(
+                'Eox-tenant had an error loading %s. Directory does not exist',
+                microsites_root
+            )
+
+    def set_key_to_cache(self, key, value):
+        """
+        Stores a key value pair in a cache scoped to the thread
+        """
+        if hasattr(self.current_request_configuration, 'cache'):
+            self.current_request_configuration.cache[key] = value
+        if not hasattr(self.current_request_configuration, 'cache'):
```

Comment:
Think this could be an elif instead of if not? @felipemontoya

Reply:
Nice catch. This logic actually looks weird now. I'm going to review it in more detail.